api/api_sources/sources/database/database-schema/schema-sqls/SoilTextureCodeSchema.sql

```diff
@@ -0,0 +1,39 @@
+-- ### Creating Table: soil_texture_code ### --
+
+        
+CREATE TABLE soil_texture_code ();
+ALTER TABLE soil_texture_code ADD COLUMN soil_texture_code_id SERIAL PRIMARY KEY;
+ALTER TABLE soil_texture_code ADD COLUMN soil_texture_code VARCHAR(1) NOT NULL UNIQUE;
+ALTER TABLE soil_texture_code ADD COLUMN description VARCHAR(100) NULL;
+ALTER TABLE soil_texture_code ADD COLUMN active_ind BOOLEAN NOT NULL DEFAULT TRUE;
+
+
+        
+-- ### Creating Comments on table ### --
+
+        
+COMMENT ON TABLE soil_texture_code IS 'Code table for schema of soil texture codes';
```

Comment:
I'd like to recommend we add a more descriptive table comment that describes "soil texture code" from a business perspective.

For example (from IAPP); The soil texture code provides details regarding the quality and water-retention of soil in a site where an invasive plant has been identified. Values are C (Coarse), M (Medium), F (Fine), O (Organic).